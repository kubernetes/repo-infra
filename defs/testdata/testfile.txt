This is just some data to hash
