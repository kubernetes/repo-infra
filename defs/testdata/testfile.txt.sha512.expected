77f3b6ab7f0b30eac6c84baecd7308c462df575b5a1bea484253fcdf5a0f61f0d9aeaa5a25f51a1621c90d0cc461f041ba5dba7573092221287af620f0f1c573
